<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="6" hostname="DESKTOP-VQKRNNU" failures="1" tests="3" name="Test2" time="0.307" errors="0" timestamp="2023-01-04T13:38:57 IST">
  <testcase classname="Hari_Practice.Assert_XML" name="SoftAssert2" time="0.132">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	xyz expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
xyz expected [true] but found [false]
at Hari_Practice.Assert_XML.SoftAssert2(Assert_XML.java:70)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- SoftAssert2 -->
  <testcase classname="Hari_Practice.testngpracticeTest" name="AAA" time="0.011"/>
  <testcase classname="Hari_Practice.testngpracticeTest" name="CCC" time="0.002"/>
  <testcase classname="Hari_Practice.testngpracticeTest" name="GGG" time="0.0">
    <ignored/>
  </testcase> <!-- GGG -->
  <testcase classname="Hari_Practice.testngpracticeTest" name="BBB" time="0.0">
    <ignored/>
  </testcase> <!-- BBB -->
  <testcase classname="Hari_Practice.testngpracticeTest" name="DDD" time="0.0">
    <ignored/>
  </testcase> <!-- DDD -->
  <testcase classname="Hari_Practice.Assert_XML" name="HardAssert1" time="0.0">
    <ignored/>
  </testcase> <!-- HardAssert1 -->
  <testcase classname="Hari_Practice.testngpracticeTest" name="FFF" time="0.0">
    <ignored/>
  </testcase> <!-- FFF -->
  <testcase classname="Hari_Practice.testngpracticeTest" name="EEE" time="0.0">
    <ignored/>
  </testcase> <!-- EEE -->
</testsuite> <!-- Test2 -->
